src/test/scala/uk/gov/hmrc/agentmtdidentifiers/model/UtrSpec.scala

```diff
@@ -36,11 +47,15 @@ class UtrSpec extends FlatSpec with Matchers {
     Utr.isValid("200000000B") shouldBe false
   }
 
+  it should "be false when it us akk non-digit characters" in {
```

Comment:
What value does this add given the "be false when it has non-digit characters" case above?